Gemfile.lock

```diff
@@ -179,7 +179,7 @@ GEM
       pry (~> 0.10)
     public_suffix (4.0.1)
     puma (3.12.4)
-    rack (2.0.8)
+    rack (2.2.2)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails (5.1.7)
```

Comment:
The Rails version is staying the same?

Reply:
:sheep: Probably should have looked at that console output a little closer.